praw/models/reddit/submission.py

```diff
@@ -31,25 +31,30 @@ class Submission(RedditBase, SubmissionListingMixin, UserContentMixin):
                              `Unix Time`_.
     ``distinguished``        Whether or not the submission is distinguished.
     ``edited``               Whether or not the submission has been edited.
-    ``id``                   The ID of the submission.
-    ``is_video``             Whether or not the submission is a Reddit-hosted
-                             video.
-    ``link_flair_css_class`` The CSS class for the submissions' flair.
-    ``link_flair_text``      The flair text for the submissions' flair.
+    ``id``                   ID of the submission.
+    ``is_self``              Whether or not the submission is a selfpost
+                             (text-only).
+    ``link_flair_template_id`` The link flair's ID, or None if not flaired.
```

Comment:
I think the entire table has to be adjusted due to the length of this field. Hence why the tests are currently failing.